src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue23563.cs

```diff
@@ -0,0 +1,25 @@
+﻿#if ANDROID
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue23563 : _IssuesUITest
+	{
+		public Issue23563(TestDevice testDevice) : base(testDevice)
+		{
+		}
+
+		public override string Issue => "Unable to set toolbar overflow menu color if not using shell";
+
+		[Test]
+		[Category(UITestCategories.Navigation)]
+		public void OverflowIconShouldBeRed()
+		{
+			App.WaitForElement("Label");
+			VerifyScreenshot();
```

Comment:
I know is an Android bug, but, to validate and align the behavior between all platforms, I think this test have sense to run everywhere.

Reply:
Yea, I disabled it because iOS doesn't have the overflow icon and I don't know how Windows behaves
![Simulator Screenshot - iPhone 16 Pro Max - 2024-10-31 at 14 25 20](https://github.com/user-attachments/assets/7617f4d6-48fb-473b-aa2a-4f0b47d4da1f)